web/app/js/components/StatusTable.jsx

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
 import * as d3 from 'd3';
 import { Link } from 'react-router-dom';
-import { Table, Tabs } from 'antd';
+import { Table, Tabs, Tooltip } from 'antd';
 import { toClassName, metricToFormatter } from './util/Utils.js';
 
 const getStatusDotCn = status => {
   return `status-dot-${status === "good" ? "green" : "grey"}`;
 }
 
+const statusDotExplanation = {
+  good: "has been added to the mesh",
```

Comment:
This message makes sense for the data plane section, but I think we might want a different message for the pods in the control plane section. Maybe something along the lines of "is up and running" or something like that.

Reply:
Sounds good! I've changed it to use "is up and running" and "has not been started" for the control plane message.